db/migrate/20220407210610_create_server_group.rb

```diff
@@ -0,0 +1,22 @@
+class ServerGroup < ApplicationRecord
+  def change
+    create_table :server_groups do |t|
+      t.string :name
+      t.string :policy
+      t.string :ems_ref
+      t.string :type
+
+      t.references :availability_zone, :type => :bigint
+      t.references :cloud_tenant,      :type => :bigint
+      t.references :ems,               :type => :bigint
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
all of this goes into the migration.

copy an paste the `create_table .. end` down to the Migration's `change`